src/OutputPanel.vala

```diff
@@ -150,7 +175,13 @@ public class Sound.OutputPanel : Gtk.Grid {
         attach (audio_alert_check, 1, 4);
         attach (visual_alert_check, 2, 4);
         attach (alerts_info, 1, 5, 2);
-        attach (test_button, 0, 6, 4);
+        attach (screen_reader_label, 0, 6, 1, 1);
+        attach (screen_reader_switch, 1, 6, 1, 1);
+        attach (screen_reader_description_label, 1, 7, 1, 1);
```

Comment:
width and height are implicitly `1` already, so you don't need to include them unless you're setting it to something other than `1`. Also, the switch and description label here should have a greater width so as not to break the layout of the checkboxes above:

```suggestion
        attach (screen_reader_label, 0, 6);
        attach (screen_reader_switch, 1, 6, 2);
        attach (screen_reader_description_label, 1, 7, 2);
```